.github/workflows/ci-dgraph-ldbc-tests.yml

```diff
@@ -0,0 +1,71 @@
+name: ci-dgraph-ldbc-tests
+on:
+  push:
```

Comment:
@anurags92 we may want to schedule it every night if this test is really heavy from a resource and time perspective. It need not run on every PR. It would be nice if we could.

Reply:
@akon-dey this is not resource intensive because we are using `ubuntu-latest` (which is a FREE runner from `GitHub`). This setting is on line#13 https://github.com/dgraph-io/dgraph/pull/8367/files#diff-f0eb7c988e7ed7e3763ebd22f0f52e899c5182c5385b2dfa1a504db51ff88841R13